### PR TITLE
Add the return type to function comments

### PR DESCRIPTION
**Pull request by TheMY3**
Hello @tobyS 
As you said in https://github.com/tobyS/pdv/issues/10 I created a pull request for return types.

It works fine for this functions like this:
image
image
image

### CODE REVIEW

**Comment by kubaceg** [comment]
:+1:  it works like a charm :) Thanks

**Review by kubaceg — APPROVED** [review]

**Comment by seagoj** [comment]
No chance you could pull in these changes? I'd really like to use them.

**Comment by gvanbeck** [comment]
+1

**Comment by esbenboye** [comment]
I know this is an old pull request, but it would make this plugin much more usefull if this was merged in!

**Comment by mudrd8mz** [comment]
+1

**Comment by mudrd8mz** [comment]
Works like a charm, thanks @YaroslavMolchan !

File: templates/function.tpl [hunk 1]
```diff
@@ -2,4 +2,5 @@
  * {{name}}
  *{{#parameters}}
  * @param {{type}}{{^type}}mixed{{/type}} ${{name}}{{/parameters}}
- */
+ {{#return}}* @return {{return}}
```
Comment (mudrd8mz) [inline templates/function.tpl:5]: Let me suggest to change this to
```
{{#return}}* @return ${{{?func: vmustache#IncrementCounter("params")}}:{{return}}} ${{{?func: vmustache#IncrementCounter("params")}}}
```
to define additional two tab-stops and have the behaviour identical with tab-stops on the `@param` lines.

**Pull request closed without merging**